code/modules/mob/living/silicon/robot/robot_modules.dm

```diff
@@ -161,6 +161,7 @@ obj/item/weapon/robot_module/proc/fix_modules() //call this proc to enable click
 	src.modules += new /obj/item/weapon/surgicaldrill(src)
 	src.modules += new /obj/item/weapon/revivalprod(src)
 	src.modules += new /obj/item/weapon/crowbar(src)
+	src.modules += new /obj/item/weapon/inflatable_dispenser/robot
```

Comment:
Medborgs are basically paramedic+, they should be able to get humans out of breached areas without killing them harder with the air.